spec/models/article_spec.rb

```diff
@@ -7,8 +7,13 @@
     end
   end
 
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :title }
+    it { is_expected.to validate_presence_of :body }
```

Comment:
Now that you have these validations in place. I would like to see some acceptance tests for what happens IF the user fails to provide values. Add them for the edit functionality and add a ticket on your PT for create.

Reply:
Sad Path in place!

Reply:
👍🏼